src/commands/org/create/scratch.ts

```diff
@@ -91,6 +91,13 @@ export default class OrgCreateScratch extends SfCommand<ScratchCreateResponse> {
       },
       helpGroup: definitionFileHelpGroupName,
     }),
+    snapshot: Flags.string({
+      char: 's',
+      summary: messages.getMessage('flags.snapshot.summary'),
+      description: messages.getMessage('flags.snapshot.description'),
+      exclusive: ['edition'],
```

Comment:
Should this also be exclusive with `source-org` (as mentioned above in the help)?